package.json

```diff
@@ -37,66 +37,66 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~9.1.0",
-    "@angular/cdk": "^9.0.1",
-    "@angular/common": "~9.1.0",
-    "@angular/compiler": "~9.1.0",
-    "@angular/core": "~9.1.0",
-    "@angular/flex-layout": "^9.0.0-beta.29",
-    "@angular/forms": "~9.1.0",
-    "@angular/material": "^9.0.1",
-    "@angular/platform-browser": "~9.1.0",
-    "@angular/platform-browser-dynamic": "~9.1.0",
-    "@angular/router": "~9.1.0",
+    "@angular/animations": "^9.1.9",
+    "@angular/cdk": "^9.2.4",
+    "@angular/common": "^9.1.9",
+    "@angular/compiler": "^9.1.9",
+    "@angular/core": "^9.1.9",
+    "@angular/flex-layout": "^9.0.0-beta.31",
+    "@angular/forms": "^9.1.9",
+    "@angular/material": "^9.2.4",
+    "@angular/platform-browser": "^9.1.9",
+    "@angular/platform-browser-dynamic": "^9.1.9",
+    "@angular/router": "^9.1.9",
     "@ngx-translate/core": "^12.1.2",
     "@ngx-translate/http-loader": "^4.0.0",
-    "@nrwl/angular": "^9.1.2",
+    "@nrwl/angular": "^9.3.0",
     "@swimlane/ngx-charts": "^13.0.2",
     "@swimlane/ngx-graph": "^7.0.0",
-    "apollo": "^2.22.0",
     "angular-crumbs": "^3.0.1",
+    "apollo": "^2.28.0",
     "apollo-angular": "^1.8.0",
     "apollo-angular-link-http": "^1.9.0",
-    "apollo-cache-inmemory": "^1.6.5",
-    "apollo-client": "^2.6.8",
-    "bootstrap": "^4.4.1",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
+    "bootstrap": "^4.5.0",
     "chart.js": "^2.9.3",
     "core-js": "^3.6.4",
-    "d3-graphviz": "^3.0.5",
+    "d3-graphviz": "^3.1.0",
     "d3-shape": "^1.3.7",
     "graphql": "^15.0.0",
     "graphql-tag": "^2.10.3",
-    "mat-table-exporter": "^9.0.0",
-    "moment-timezone": "^0.5.28",
-    "ng2-charts": "^2.3.0",
+    "mat-table-exporter": "^9.0.2",
+    "moment-timezone": "^0.5.31",
+    "ng2-charts": "^2.3.2",
     "oidc-client": "^1.10.1",
     "rxjs": "~6.5.4",
     "save": "^2.4.0",
     "tinycolor2": "^1.4.1",
-    "tslib": "^1.11.0",
+    "tslib": "^1.13.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.901.1",
     "@angular/cli": "9.1.1",
-    "@angular/compiler-cli": "~9.1.0",
-    "@angular/language-service": "~9.1.0",
-    "@nrwl/cypress": "^9.1.2",
+    "@angular/compiler-cli": "^9.1.9",
+    "@angular/language-service": "^9.1.9",
+    "@nrwl/cypress": "^9.3.0",
     "@nrwl/jest": "9.2.2",
     "@nrwl/workspace": "9.2.2",
-    "@openapitools/openapi-generator-cli": "^1.0.13-4.2.2",
+    "@openapitools/openapi-generator-cli": "^1.0.13-4.3.1",
```

Comment:
Please, try if after this update the generated libraries are ok. If not, downgrade to the original version.